src/gatling/scala/uk/gov/hmcts/reform/cmc/performance/simulations/CreateClaimSimulation.scala

```diff
@@ -41,7 +41,7 @@ class CreateClaimSimulation extends Simulation with SimulationHooks {
       )
 
   setUp(createClaimScenario
-    .inject(rampUsers(10).over(10 seconds))
+    .inject(rampUsers(1).over(10 seconds))
```

Comment:
Why is this being changed?

Reply:
@jacmurphy that has to be reverted. You should have checked it before submitting PR.

Reply:
My mistake @robertparkinson @geek-soft . Changing now